### PR TITLE
Add CAPA v26.1.0

### PR DESCRIPTION
**Pull request by nprokopic**
Towards https://github.com/giantswarm/roadmap/issues/3610

This release updates the apps and components, keeping them up to date with the latest v25 release. It also brings improvements for the container registry usage.

#### Changed

- All workload clusters will by default use Zot registry as a pull-through cache of Azure Container Registry.

#### Fix
- Remove quotes from acme-http01-solver-image argument. The quotes are used when looking up the image which causes an error.

#### Update
- Improves container security by setting `runAsGroup` and `runAsUser` greater than zero for all deployments.

### containerlinux [3815.2.5](https://www.flatcar-linux.org/releases/#release-3815.2.5)

_Changes since **Stable 3815.2.4**_

#### Security fixes:

- openssh ([CVE-2024-6387](https://nvd.nist.gov/vuln/detail/CVE-2024-6387))

#### Updates:

- Linux ([6.1.96](https://lwn.net/Articles/979851))
- openssh ([9.7_p1](https://www.openssh.com/txt/release-9.7))

#### Changed
- Fix regression setting Policy BPF Max map policyMapMax back to 65536 from 16384.
- Upgrade cilium to v1.15.6.

## Checklist
- [x] Roadmap issue created
- [x] Release uses latest stable Flatcar
- [x] Release uses latest Kubernetes 1.26.x patch version

## Triggering e2e tests

To trigger the E2E test for each new Release added in this PR add a comment with the following:

`/run releases-test-suites`

If you want to trigger conformance tests you can do so by adding a comment similar to the following:

`/run conformance-tests RELEASE_VERSION=v28.0.0 PROVIDER=capa`

For more details see the [README.md](/README.md#running-tests-against-prs)

### CODE REVIEW

**Comment by nprokopic** [comment]
/run releases-test-suites TARGET_SUITES=./providers/capa/standard

**Comment by nprokopic** [comment]
Debugging e2e tests

/run releases-test-suites TARGET_SUITES=./providers/capa/standard

**Comment by nprokopic** [comment]
e2e tests have failed because the AMI does not exist. v26.1.0 changes Flatcar version, while keeping the same k8s version, but the new image does not exist as the previous one has been also built manually. Will trigger a new image-build for this.

**Comment by nprokopic** [comment]
/run releases-test-suites TARGET_SUITES=./providers/capa/standard

**Comment by nprokopic** [comment]
Testing minor upgrade 26.0.0 to 26.1.0.

/run releases-test-suites TARGET_SUITES=./providers/capa/upgrade PREVIOUS_RELEASE=26.0.0

**Comment by nprokopic** [comment]
Testing major upgrade 25.0.0 to 26.0.0.

/run releases-test-suites TARGET_SUITES=./providers/capa/upgrade PREVIOUS_RELEASE=25.0.0

**Review by Gacko — APPROVED** [review]

File: capa/v26.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+- Linux ([6.1.96](https://lwn.net/Articles/979851))
+- openssh ([9.7_p1](https://www.openssh.com/txt/release-9.7))
+
+### cilium [0.25.1](https://github.com/giantswarm/cilium-app/releases/tag/v0.25.1)
+
+#### Changed
+- Fix regression setting Policy BPF Max map policyMapMax back to 65536 from 16384.
+- Upgrade cilium to v1.15.6.
+
```
Comment (Gacko) [inline capa/v26.1.0/README.md:39]:
```suggestion
```

File: capa/v26.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- openssh ([CVE-2024-6387](https://nvd.nist.gov/vuln/detail/CVE-2024-6387))
+
+#### Updates:
+
+- Linux ([6.1.96](https://lwn.net/Articles/979851))
+- openssh ([9.7_p1](https://www.openssh.com/txt/release-9.7))
+
+### cilium [0.25.1](https://github.com/giantswarm/cilium-app/releases/tag/v0.25.1)
+
+#### Changed
```
Comment (Gacko) [inline capa/v26.1.0/README.md:36]:
```suggestion
#### Changed

```

File: capa/v26.1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+- All workload clusters will by default use Zot registry as a pull-through cache of Azure Container Registry.
+
+### cert-manager [3.7.9](https://github.com/giantswarm/cert-manager-app/releases/tag/v3.7.9)
+
+#### Fix
+- Remove quotes from acme-http01-solver-image argument. The quotes are used when looking up the image which causes an error.
+
+#### Update
```
Comment (Gacko) [inline capa/v26.1.0/README.md:18]:
```suggestion
#### Update

```

File: capa/v26.1.0/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+### cluster-aws [1.3.0](https://github.com/giantswarm/cluster-aws/releases/tag/v1.3.0)
+
+#### Changed
+
+- All workload clusters will by default use Zot registry as a pull-through cache of Azure Container Registry.
+
+### cert-manager [3.7.9](https://github.com/giantswarm/cert-manager-app/releases/tag/v3.7.9)
+
+#### Fix
```
Comment (Gacko) [inline capa/v26.1.0/README.md:15]:
```suggestion
#### Fix

```

**Comment by nprokopic** [comment]
@T-Kukawka @weatherhog @gawertm please check the announcement and README for v26.1.0. It mentions a new cluster-aws version with Zot changes, as well as other app/component changes that your teams own and we have already included in v25.1.0.

v26.1.0 has been tested and can be merged after this PR has been approved.

**Review by T-Kukawka — APPROVED** [review]

**Review by tuladhar — APPROVED** [review]

**Review by mcharriere — APPROVED** [review]

**Pull request merged**